lonestarmine/include/lonestarmine.h

```diff
@@ -3,7 +3,7 @@
 #include <sstream>
 #include <iostream>
 #include "llvm/Support/CommandLine.h"
-#ifndef ENABLE_GPU
+#ifndef __GALOIS_HET_CUDA__
```

Comment:
Tangential to this, but we generally should avoid underscore prefixes on preprocessor symbols since that class of names is supposed to be reserved for the compiler. https://stackoverflow.com/a/228797 has details.

Reply:
Yeah totally. I'd prefer a different name for this macro myself.